python/requirements.txt

```diff
@@ -1,7 +1,7 @@
-jinja2 == 3.1.3
-numpy == 1.24.4
-pandas == 2.0.3
-pytest == 8.1.1
-coverage == 7.4.4
+jinja2 == 3.1.4
+numpy == 2.0.2
+pandas == 2.2.3
+pytest == 8.3.3
+coverage == 7.6.1
 pytest-cov == 5.0.0
-# pytest-xdist == 3.5.0
+# pytest-xdist == 3.6.1
```

Comment:
So, I bumped this deps. Really the only needed were pandas and numpy...

I picked a numpy recent enough that it:
* Supports Python 3.12
* Does not drop support for Python 3.9 yet

But that means it drops support for python 3.8